packages/common/components/content/blocks/page-header.marko

```diff
@@ -1,11 +1,24 @@
 import { getAsObject } from '@base-cms/object-path';
 
 $ const content = getAsObject(input, 'content');
+$ const specialTypes = ['event', 'webinar', 'whitepaper'];
```

Comment:
I don't believe the `whitepaper` content type utilizes start/end dates, correct?